docs/deploy/rock-on-rhel.md

```diff
@@ -0,0 +1,185 @@
+```
+
+if it returns `0` then it has been properly disabled
+
+
+
+## Deployment of Rock across All Machines
+Mount the latest iso in this case its rocknsm2-4.iso to `/mnt`
+
+Copy the folders form the mounted device to `/srv/rocknsm`
+
+Create the a place for your pet ROCK to live
+```
+sudo mkdir -p /usr/share/rock
+```
+
+Git Clone or Copy the rocknsm repo to the `/usr/share/rock` directory
+```
+sudo git clone https://github.com/rocknsm/rock.git
+```
+
+Install Ansible to coordinate the installation of the Sensor
+```
+sudo yum install ansible
+```
+
+Ensure the latest version of markupsafe is installed also
+```
+sudo yum install python2-markupsafe
+```
+
+Copy the hosts.ini file that so ansible knows where to deploy things using `sudo cp /usr/share/rock/etc/rocknsm/hosts.ini /etc/rocknsm/.`
+
+> NOTE: Most of the Rock configuration is now automated and can be called from anywhere on the os.
+
+```
+[admin@sensor ~]$ sudo rock help
+Usage: /sbin/rock COMMAND [options]
+Commands:
+setup               Launch TUI to configure this host for deployment
+tui                 Alias for setup
+ssh-config          Configure hosts in inventory to use key-based auth (multinode)
+deploy              Deploy selected ROCK components
+deploy-offline      Same as deploy --offline (Default ISO behavior)
+deploy-online       Same as deploy --online
+stop                Stop all ROCK services
+start               Start all ROCK services
+restart             Restart all ROCK services
+status              Report status for all ROCK services
+genconfig           Generate default configuration based on current system
+destroy             Destroy all ROCK data: indexes, logs, PCAP, i.e. EVERYTHING
+                      NOTE: Will not remove any services, just the data
+
+Options:
+--config, -c <config_yaml>         Specify full path to configuration overrides
+--extra, -e <ansible variables>    Set additional variables as key=value or YAML/JSON passed to ansible-playbook
+--help, -h                         Show this usage information
+--inventory, -i <inventory_path>   Specify path to Ansible inventory file
+--limit <host>                     Specify host to run plays
+--list-hosts                       Outputs a list of matching hosts; does not execute anything else
+--list-tags                        List all available tags
+--list-tasks                       List all tasks that would be executed
+--offline, -o                      Deploy ROCK using only local repos (Default ISO behavior)
+--online, -O                       Deploy ROCK using online repos
+--playbook, -p <playbook_path>     Specify path to Ansible playbook file
+--skip-tags <tags>                 Only run plays and tasks whose tags do not match these values
+--tags, -t <tags>                  Only run plays and tasks tagged with these values
+--verbose, -v                      Increase verbosity of ansible-playbook
+```
+
+Change Directory into `usr/share/rock/bin` if you are not already there

```

Comment:
This is where Rock should be installed as an RPM since it does distribute a rock.spec file instead of forcing the user to CD into /usr/share/...